### DISCUSSION TITLE
door-pwoerr joint axis

### DISCUSSION THREAD

**Discussion by MhdMohammadi** [StanfordVL/BEHAVIOR-1K#1098] [General]
Hi, it seems `door_pwoerr` has a joint, attached in a wrong direction, as it opens in a weird way. [Here is the link](https://behavior.stanford.edu/knowledgebase/objects/door-pwoerr/index.html) of the door in the knowledgebase, and an image of open state is attached. 
![Screenshot 2024-12-17 175537](https://github.com/user-attachments/assets/9b9a338e-5ba9-4b51-914a-6da1974ef37d)

**Comment by hang-yin** [comment]
@cgokmen

**Comment by cgokmen** [comment]
Acknowledged, will be fixed in our next released. Thanks for the report.
